Makefile.devcontainer

```diff
@@ -52,14 +71,12 @@ fuzz-grafana:                   ## Run fuzzer for services/grafana package.
 
 check:                          ## Run required checkers and linters.
 	go run .github/check-license.go
-	dep check
-	go-sumtype ./vendor/... ./...
```

Comment:
we should still keep this check.

Reply:
how? we removed vendor folder.
We removed this check from previos repositories

Reply:
there are 2 folders :)

Reply:
Do we use it somewhere? 
Maybe I don't get usage, but based on README from go-sumtype
Comment is needed
```
//go-sumtype:decl MySumType
```
And I can't see it anywhere in our codebase

Reply:
so I checked and we need it, those comments are only there in pmm repo because the types for which go-sumtype is used for are in the pmm repo, but these types are used in switch cases for example

`inventorypb.Node` type in pmm repo has the directive comment  https://github.com/percona/pmm/blob/b23824e4facf723fdbb08b0ea28f30c9f51e386e/api/inventorypb/nodes.go#L3

and the switch case for that is in pmm-managed here https://github.com/percona/pmm-managed/blob/PMM-2.0/services/management/management.go#L100

so go-sumtype is needed in this repo to check for exhaustivness of such types and there are many such types, so we should keep it

Reply:
sorry the second link was wrong, here is the usage of the NodeTypes in switch statements  https://github.com/percona/pmm-managed/blob/PMM-2.0/services/inventory/grpc/nodes_server.go#L67-L78

if you remove any one case from it then go-sumtype would fail  

```
/root/go/src/github.com/percona/pmm-managed/services/inventory/grpc/nodes_server.go:66:3: exhaustiveness check failed for sum type 'Node': missing cases for RemoteAzureDatabaseNode
make[1]: *** [check] Error 1
```

so it works and we should keep it

Reply:
now that we are removing vendor we should change the path for the dependency we give in the go-sumtype command `go-sumtype ./...  <path-to-dependencies>`

Reply:
Is it needed to check depedencies? If yes, how to do it. It isn't straightforward

Reply:
yes the dependency (or the pmm repo) has the go-sumtype directive so we should supply the path to that to actually run `go-sumtype`

but that might be difficult without a vendor folder
i found this command of off the internet ` go list -m -f '{{if not (or .Indirect .Main)}}{{.Path}}{{end}}' all` which lists all modules used by a go modules project but it only lists them as import paths and not the actual paths

i guess we can skip it for now, let the command be as it is and merge this. 

I will try to find a solution for this. If I can't then we can remove it in the future. Lets merge it for now

Reply:
there is one more solution for this, that is having the `vendor` folder

Reply:
I don't want keep vendor folder just because of  linter.

Reply:
If there is no other way, let's go forward without this linter.

Reply:
`A simple utility for running exhaustiveness checks on type switch statements.`

As I understand this linter checks that all options specified in switch statement. So in dependencies we have types that "prepared" for that, but in this repo we have switch statements. So we don't need vendor or something else, this linter will work for that repo.